### ISSUE TITLE
Обратная совместимость со старыми браузерами

### ISSUE THREAD

**Issue by maksalees** [OpenVK/openvk#362]
Из названия думаю все понятно
- [ ] Компилировать через Babel весь JavaScript
- [ ] Использовать autoprefixer и postcss-preset-env

**Comment by SigmaTel71** [comment]
> Убрать использование новых функций (flexbox и прочие) из темплейтов и CSS

А что не так с Flexbox? Отсутствие поддержки в IE9 и ниже? Сначала бы понять, какая «целевая» платформа у основного стиля OpenVK. Поддержка Flexbox постепенно проявляется ещё в ранних версиях Firefox через vendor-свойство [-moz-box-flex](https://developer.mozilla.org/en-US/docs/Web/CSS/box-flex). Аналогично для Chrome/Chromium, только -webkit. Если верить блогу Opera, то он есть в [Presto 12.1+](https://dev.opera.com/articles/advanced-cross-browser-flexbox/).

**Comment by ghost** [comment]
Стоит использовать autoprefixer и postcss-preset-env.

**Comment by SigmaTel71** [comment]
> Стоит использовать autoprefixer и postcss-preset-env.

_фу-фу-фу, лишний JS ещё тащить_

**Comment by ghost** [comment]
> _фу-фу-фу, лишний JS ещё тащить_

Это не JS, а CSS.

**Comment by SigmaTel71** [comment]
> Это не JS, а CSS.

Чойта вдруг? Когда это транспайлеры стали писать не на JS, а на CSS?

**Comment by ghost** [comment]
> > Это не JS, а CSS.
> 
> Чойта вдруг? Когда это транспайлеры стали писать не на JS, а на CSS?

Так, я не очень понял. В чём проблема затащить транспайлер в процесс *сборки*?

**Comment by SigmaTel71** [comment]
> В чём проблема затащить транспайлер в процесс сборки?

Зачем вообще тащить транспайлер, когда все свойства с префиксами можно в отдельный CSS спихнуть (если не хочется основной захламлять) и подгружать его в зависимости от UserAgent? Ручной работы боимся?
